mobilecoind/clients/java/mob_client/src/main/proto/mobilecoind_api.proto

```diff
@@ -84,7 +83,7 @@ message AccountKey {
     string fog_fqdn = 3;
 }
 
-// A public address, used to identify receipients.
+// A public address, used to identify recipients.
```

Comment:
Err, I am only now noticing this file. It should go away or become a symlink.

Reply:
yeah, @tsegaran was planning on doing that I believe.